tests_integration/__tests__/__snapshots__/support-info.js.snap

```diff
@@ -437,6 +437,7 @@ exports[`CLI --support-info (stdout) 1`] = `
       \\"codemirrorMimeType\\": \\"text/javascript\\",
       \\"codemirrorMode\\": \\"javascript\\",
       \\"extensions\\": [
+        \\".jsonc\\",
```

Comment:
Do we want use yaml parser for `.jsonc`?

Reply:
This change has nothing to do with YAML.

Reply:
Sorry, missed some context.